.github/workflows/main.yml

```diff
@@ -5,7 +5,14 @@ concurrency:
   cancel-in-progress: true
 
 on:
-  pull_request:
+  pull_request:  # TODO: remove after merge BI-5174
```

Comment:
otherwise CI wont start for current PR since pull_request_target trigger uses main branch version of workflow